src/Pos/Update/Poll/Logic/Apply.hs

```diff
@@ -294,9 +294,7 @@ applyDepthCheck hh cd
     | otherwise = do
         deepProposals <- getDeepProposals (cd - blkSecurityParam)
         let winners =
-                concatMap toList $
-                map resetAllDecisions $
-                map (NE.sortBy proposalCmp) $
+                concatMap ((toList . resetAllDecisions) . NE.sortBy proposalCmp) $
```

Comment:
Brackets wrap `toList . resetAllDecisions` to warm it up. We were using redundant brackets in winter when it was cold. Now it's getting warmer, so it's not necessary.

Nevermind, local meme 😜